gcloud/storage/connection.py

```diff
@@ -27,6 +26,7 @@
 from oauth2client import crypt
 from oauth2client import service_account
 import pytz
+from six.moves.urllib.parse import urlencode  # pylint: disable=F0401
```

Comment:
I'm fine with leaving comments in our code to disable buggy checks in the tool, where the tool's own config can't disable it.